dist/tools/doccheck/check.sh

```diff
@@ -74,3 +76,47 @@ then
     echo "${UNDEFINED_GROUPS_PRINT}"
     exit 2
 fi
+
+# Check for groups defined multiple times
+MULTIPLE_DEFINED_GROUPS=$( \
+    for group in $(echo ${DEFINED_GROUPS_UNIQUE} | tr ' ' '\n'); \
```

Comment:
Would a `echo ${DEFINED_GROUPS} | sort | uniq -c` give a good enough output ?

Reply:
I tried it and could make it work.